manage: add inverted filters (disabled, external)

tophf opened the pull request:
The checkbox filters can be inverted now.
A bit unusual, though, since the entire label is a selector.

I've tried and discarded a couple of alternative solutions, and this one seems the least cluttered and most self-evident to me. Not sure about convenience as I almost never use the filters.

Please test: [invert-filters.zip](https://github.com/openstyles/stylus/archive/invert-filters.zip).

![clip340-fs8](https://user-images.githubusercontent.com/1310400/29533127-296bf6f0-86b9-11e7-8c23-744cf74e16c1.png)

narcolepticinsomniac commented:
Functionality is good, and useful to some I'm sure. The lack of width adjustment and therefore arrow position is weird when switching back and forth between local and external filters. Either different wording with a fixed width, or a self-adjusting width would be better.

![select](https://user-images.githubusercontent.com/14100003/29542346-6e028092-86a8-11e7-823c-5c9792333374.png)

Mottie commented:
Maybe we could adapt Google's material design like in [Materialize](http://materializecss.com/forms.html#select)?

![](https://user-images.githubusercontent.com/136959/29542762-ce83c98c-86a1-11e7-9841-5416fd043ef6.png)

tophf commented:
* Added autosizing because an arrow immediately to the right is more obvious, hopefully.
* Added early initialization of all option elements to prevent page flicker while loading, which was especially weird when the wrong inverted filter label or the wrong style list results were visible briefly.

tophf commented:
If there are no more suggestions I'll merge it because it doesn't create conflicts with other PRs.

Mottie commented:
LGTM!
